code/modules/research/designs/machine_designs.dm

```diff
@@ -1347,3 +1347,13 @@
 		RND_CATEGORY_MACHINE + RND_SUBCATEGORY_MACHINE_ENGINEERING
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_ENGINEERING | DEPARTMENT_BITFLAG_CARGO
+
+/datum/design/board/mailsorter
+	name = "Mail Sorter"
+	desc = "The circuit board for a mail sorting unit."
+	id = "mailsorter"
```

Comment:
This ID doesn't show up in any research node so it's unprintable.

Reply:
My bad, will do that now